flattentool/schema.py

```diff
@@ -124,7 +124,23 @@ def parse_schema_dict(self, parent_path, schema_dict, parent_id_fields=None, tit
             parent_path = parent_path + '/'
         parent_id_fields = parent_id_fields or []
         title_lookup = self.title_lookup if title_lookup is None else title_lookup
-        if 'properties' in schema_dict:
+
+        if 'type' in schema_dict and schema_dict['type'] == 'array' \
+                and 'items' in schema_dict and 'oneOf' in schema_dict['items']:
+            for oneOf in schema_dict['items']['oneOf']:
+                if oneOf['type'] == 'object':
```

Comment:
Just one more thing, a one of clause does not have to have a type so you will need to check if type exists first.

Reply:
Oh yes, thanks